modules/govuk_jenkins/templates/jobs/deploy_app.yaml.erb

```diff
@@ -42,6 +42,27 @@
 
             git clone ${APP_DEPLOYMENT_GIT_URL} --branch master --single-branch --depth 1 ./
             ./jenkins.sh
+        - conditional-step:
```

Comment:
It might be kind to future developers to put a comment above this line along the lines of

```
        # Only deploy if this is a "deploy" task, release tag and a supported app
        # https://docs.openstack.org/infra/jenkins-job-builder/builders.html#builders.conditional-step
        - conditional-step:
```

Reply:
Sure, I've added comments for each of the conditions.